source/index.js

```diff
@@ -2,7 +2,12 @@
 const pkg = require('../package.json');
 const create = require('./create');
 
-const defaults = {
+// @todo
+// I exported this default to be able to infer typings from it.
+// I'm guessing this `defaults` is being merged with user options
+// so `as-stream` receives an object that has merged props.
+// Let's get a confirmation from a maintainer but I can also create an explicit typings for it.
+export const defaults = {
```

Comment:
Like the comment says, I can also create an explicit typing for the `defaults` value. I use this value as a type directly using `typeof` operator.

I just wanted to make sure:
A) Are you guys okay with this, or would you prefer an explicit Default typing
B) Am I correct to assumre that `as-stream` receives a merged object from default and passed in options?